index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Google's clon</title>
+    <link rel="stylesheet" href="./reset.css">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="topnav">
+                <a class="active" href="#about">About</a>
+                <a href="#store">Store</a>
+                <div class="topnav-right">
+                    <a href="#gmail">Gmail</a>
+                    <a href="#images">Images</a>
+                    <a rel="noopener" href="#"><span class="material-icons">apps</span></a>
+                    <a rel="noopener" href="https://github.com/micheaol" target="_blank"> <span class="material-icons">account_circle</span></a>
+                </div>
```

Comment:
It would be better to use an unordered list here for each element ul > li. This will be more semantically correct